felix/bpf-gpl/policy_program.h

```diff
@@ -72,6 +72,7 @@ int calico_tc_norm_pol_tail(struct __sk_buff *skb)
 	state->pol_rc = execute_policy_norm(skb, state->ip_proto, state->ip_src,
 					    state->ip_dst, state->sport, state->dport);
 
+	CALI_DEBUG("jumping to allowed\n");
```

Comment:
Nit: would it help to add a parameter to what the "jumping is allowed to" e.g. `state->ip_dst` or whatever make sense?

Reply:
it is not the full context anyway, there might be a way how to fix it in general, but that would be for a different PR :)